internal/daysteps/daysteps.go

```diff
@@ -11,28 +11,27 @@ import (
 )
 
 var (
-	StepLength    = 0.65  // длина шага в метрах
-	distance      float64 //пройденная дистанция
-	ErrFormatTime = errors.New("Ошибка преобразования строки в формат времени")
-	ErrConvToInt  = errors.New("ошибка преобразования в целое число")
-	ErrLenSlice   = errors.New("len(word) < 2")
+	StepLength = 0.65  // длина шага в метрах
+	distance   float64 //пройденная дистанция
+
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// Строка в слайс строк
 	word := strings.Split(data, ",")
 	if len(word) < 2 {
-		return 0, 0, ErrLenSlice
+		err := errors.New("len(word)!=2")
+		return 0, 0, fmt.Errorf("slice length error: %w", err)
```

Comment:
Можно так не усложнять, тут нет смысла оборачивать ошибку, можно просто так сделать
```suggestion
		return 0, 0, fmt.Errorf("slice length error: len(word)!=2")
```